planning_tests/util.py

```diff
@@ -0,0 +1,25 @@
+import os
+import sys
+from unified_planning.shortcuts import OneshotPlanner, PlanValidator
+from planning_tests.correct_output import correct, correct_validation
+
+class TestUtil:
```

Comment:
I do not see the need to have a class here since you define only a static method.

Minor comment: we use four spaces for an indent.